images/techdocs/tests/test_image.py

```diff
@@ -25,6 +25,7 @@ def build_image(
         ("techdocs-cli --version", b"1.2.0"),
         ("markdownlint --version", b"0.32.2"),
         ("vale --version", b"vale version v2.20.2"),
+        ("yq --version", b"yq (https://github.com/mikefarah/yq/) version 4.28.1"),
```

Comment:
We really should not be using exact versions here, I guess it's okay since we already did, but it would be best of you only pin the major version or major/minor version so we can still get some upgrades. Like yq 4.x should really be okay still.

Reply:
I agree, at least for things post 1.0 versions. I'll make a new PR for that to get this merged.